blocks/editable/format-toolbar/index.js

```diff
@@ -182,7 +196,7 @@ class FormatToolbar extends Component {
 							href={ this.state.linkValue }
 							target="_blank"
 						>
-							{ this.state.linkValue && decodeURI( this.state.linkValue ) }
+							{ this.state.linkValue && filterURLForDisplay( decodeURI( this.state.linkValue ) ) }
```

Comment:
I wonder if we should move the `decodeURI` call inside the `filterURLForDisplay` function